nextjs/lizeth-9127/pages/index.js

```diff
@@ -0,0 +1,17 @@
+import { useState, useEffect } from "react"
+import Dog from "../components/Dog"
+
+const Home = () => {  
+const [ dogs, setDogs] = useState([])
+const getDogs = async () => {
+    const list = await fetch('/api/dogs').then(data => data.json())
+    setDogs(list)
+}
+useEffect( () => { getDogs() }, [])
+    return (
+    <>
+    <h1>Dog Listing</h1>
+    <div className="grid-wrapper">{dogs.map( dog => { return <Dog {...dog}/>})}</div>
```

Comment:
Podrias agregarle las keys para que desaparezca el warning en consola
```suggestion
    <div className="grid-wrapper">{dogs.map( (dog, index) => { return <Dog key={index} {...dog}/>})}</div>
```